File: UI/Communication/Exceptions/ServerNameAlreadyExistsException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UI.Communication.Exceptions
{
    /// <summary>
    /// Exception in case that a client try to create a server, which name is already taken
    /// </summary>
    class ServerNameAlreadyExistsException: Exception
    {
        public ServerNameAlreadyExistsException(string msg) : base(msg) { }
    }
}

File: Server/src/main/java/network/exception/ServerNameAlreadyExistsException.java
package network.exception;

/**
 * Exception which is thrown if the the name of the server already exists
 */
public class ServerNameAlreadyExistsException extends RuntimeException{
	
	private static final long serialVersionUID = -2994867398845471132L;

	public ServerNameAlreadyExistsException() {
		super();
	}
	
	public ServerNameAlreadyExistsException(String s) {
		super(s);
	}
}
